src/temp/diveButton/DiveIntoMap.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../theme/theme';
+.diveIntoMapMainContainer {
+  width: 306px;
+  height: 341px;
+  background: grey; //need to be removed
+  border-radius: 13.401px;
+  position: relative;
+}
+
+.diveIntoProjectButton {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 6px;
+  position: absolute;
+  width: 274px;
+  height: 40px;
+  background: $primaryColorNew;
+  border-radius: 8px;
+  bottom: 16px;
+  left: 16px;
```

Comment:
Absolute positioning - is that really needed? Can't we use `display: flex` and `align-items: flex-end` on the container, and margin on the button?

Absolute positioning should be a last resort if nothing else works. While it may not seem like a big difference, it can create headaches later on when adjustments need to be made.